spec/spec_helper_acceptance.rb

```diff
@@ -1,7 +1,27 @@
+# Not sure if these are still neeed.
 require 'net/http'
 require 'uri'
 require 'nokogiri'
```

Comment:
if we can get rid of nokogiri all the better, takes a long time to load

Reply:
I'm not sure if you're talking about the up-front compile time? Beaker always drags in Nokogiri anyhow as a transitive dep -> beaker-aws -> aws-sdk-v1 -> nokogiri

Does just loading the gem code also cause a hit after it's been installed?